got diffrent types of platforms in semi branch

TheDarkAngel099 opened the pull request.

yunzhouli99 committed:
Player Controller V2.0

Rewrite some code of the player controller.

kristofdanyi99 committed:
Map implemented

kristofdanyi99 committed:
Revert "Map implemented"

This reverts commit c7e06692e35128e121017f6719d33745b6dd2f0b.

bubuton committed:
added jellyFish

kristofdanyi99 committed:
Map implemented

kristofdanyi99 committed:
Update default-2022.dwlt

bubuton committed:
created jelly prefab

bubuton committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

bubuton committed:
player camera solved

yunzhouli99 committed:
Add sprint and croch

yunzhouli99 reviewed (changes requested):
All the files in the Player folder are not the latest version, better not to commit anything in that folder.

bubuton committed:
fixed enemy, added a reset function

bubuton committed:
prototype version

bubuton committed:
fixed enemy movement

TheDarkAngel099 committed:
Added Timed Platform and Vanishing platform

there is a bug with vanishing platform

TheDarkAngel099 committed:
baked navmesh

TheDarkAngel099 committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

kristofdanyi99 committed:
Vertically moving platform added

yunzhouli99 committed:
Stealth

If player is crouching on a ground whose tag is the stealth tag (have not defined yet), the tag of player will turn to "stealthPlayer" from "Player". You can compare the gameObject.tag of the player with a string like "if (player.gameObject.tag == "stealthPlayer)"

TheDarkAngel099 committed:
Added collectable Nuclear Waste

TheDarkAngel099 committed:
created prefabs

TheDarkAngel099 committed:
Fixed the waste ccollected counter

bubuton committed:
fixed AI

QizhouD committed:
Main Menu UI

bubuton committed:
final Prototype (with ui and enemy attack)

bubuton committed:
updated build

bubuton committed:
build updated

kristofdanyi99 committed:
Cleaned up hierarchy and code for vertical platform

bubuton committed:
includes read me

bubuton committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

yunzhouli99 committed:
Update PlayerController.cs

yunzhouli99 committed:
Update PlayerController.cs

TheDarkAngel099 committed:
partially added a double jump pickup

TheDarkAngel099 committed:
added double jump

kinda bugged right now as it's turned on by default. need to only enable double jump after collecting the pickup

kristofdanyi99 committed:
Getting all pickups loads level 2

level 2 not implemented yet

kristofdanyi99 committed:
.

kristofdanyi99 committed:
Update default-2022.dwlt

kristofdanyi99 committed:
Revert "Update default-2022.dwlt"

This reverts commit f9447f2ff94b871c281b9c583024b097f262a188.

kristofdanyi99 committed:
Revert "."

This reverts commit 270c6e8d8ffa34f5b4e0ae43a24efa6457e420b5.

kristofdanyi99 committed:
Revert "Getting all pickups loads level 2"

This reverts commit 1f38b257093c209dd2c7392598140aca15dbc219.

kristofdanyi99 committed:
Fixed issues with github

kristofdanyi99 committed:
Horizontally moving platform added and beginning of level 2

The platform has a bug that it doesn't carry the player. The map is not finished

kristofdanyi99 committed:
Testing blender assets

Working on assets for platforms
I made one for the spawn point, not finished

kristofdanyi99 committed:
skybox and some shading

I downloaded a skybox material. I'm applying shading to make the game look underwater (still in process)

kristofdanyi99 committed:
Skybox and ground material

kristofdanyi99 committed:
terrain and colours

I added terrain and used tetradic colours

TheDarkAngel099 committed:
updated readme to include GUI table refrance

TheDarkAngel099 committed:
Update ReadME.txt

TheDarkAngel099 committed:
Update ReadME.txt

bubuton committed:
demo of pcg

bubuton committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

TheDarkAngel099 committed:
Update ReadME.txt

yunzhouli99 committed:
fix movement's bug

TheDarkAngel099 committed:
Added PCG terrain and GUI to edit terrain values

only added PCG terrain yet to add corals and stuff

TheDarkAngel099 committed:
added terrains to every scene and textures to terrains

You can go on the terrain in one of the levels need to fix that issue potential fixes are removing collider or the player respawn when contacted with terrain

TheDarkAngel099 committed:
added a refrance

TheDarkAngel099 committed:
just in case

TheDarkAngel099 committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

bubuton committed:
fixed part of the navMesh, give terrain a layer to exclude from navMesh

TheDarkAngel099 committed:
disabled terrain coliders

ester egg exploration no more

QizhouD committed:
PlayerUI 1.0

QizhouD committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

kristofdanyi99 committed:
Platform generator v2

 it generates platforms between two game objects:
- regular
- timed
- horizontally moving
they all work as intended
positioning and sizing of the platforms  are slightly randomised.

kristofdanyi99 committed:
left out

yunzhouli99 committed:
Fixed the bug of player will silp off moving platform

yunzhouli99 committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

bubuton committed:
Update Enemy.prefab

bubuton committed:
change enemy asset to shark

bubuton committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

yunzhouli99 committed:
fix bug

bubuton committed:
fixed oxygen

bubuton committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

yunzhouli99 committed:
tagged  timeed platform

yunzhouli99 committed:
Update PlayeO2.cs

QizhouD committed:
NukeBar

QizhouD committed:
Showing Text

kristofdanyi99 committed:
Platform Generator v3

Enemies + Jellyfish + pickups added

kristofdanyi99 committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

kristofdanyi99 committed:
.

QizhouD committed:
Showing Text 2

QizhouD committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

bubuton committed:
latest demo

bubuton committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

TheDarkAngel099 committed:
changed some comments

TheDarkAngel099 committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

TheDarkAngel099 committed:
Update ReadME.txt

QizhouD committed:
OptionsMenu

TheDarkAngel099 committed:
added a refence in pcg terrain code

TheDarkAngel099 committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

QizhouD committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

QizhouD committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

bubuton committed:
Update Level_Tutorial.unity

bubuton committed:
changed bar position

bubuton committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

kristofdanyi99 committed:
Checkpoints implemented

Once you reach the next checkpoint you will respawn there

kristofdanyi99 committed:
Some level design

QizhouD committed:
Turtorial

yunzhouli99 committed:
Direction can be changed during double jumo

QizhouD committed:
Tutorial changes

QizhouD committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

TheDarkAngel099 committed:
added background sounds object

music prefab is a empty object with audio source on it. place it in scenes to have background noise /  music

QizhouD committed:
Spawn Location

QizhouD committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

QizhouD committed:
End Game UI

TheDarkAngel099 committed:
Update ReadME.txt

QizhouD committed:
Link Tutorial level

QizhouD committed:
Tutorial changes

kristofdanyi99 committed:
Levels finished

kristofdanyi99 committed:
Merge branch 'main' of https://github.com/MPGD-Group-B/Prototype

QizhouD committed:
Timer

bubuton committed:
new build

bubuton committed:
updated build

bubuton committed:
Updated ReadMe

bubuton committed:
fixed tutorial build
